pymc3/tests/test_quadpotential.py

```diff
@@ -271,7 +272,7 @@ def test_full_adapt_sampling(seed=289586):
     L[np.triu_indices_from(L, 1)] = 0.0
 
     with pymc3.Model() as model:
-        pymc3.MvNormal("a", mu=np.zeros(len(L)), chol=L, shape=len(L))
+        pymc3.MvNormal("a", mu=np.zeros(len(L)), chol=L, size=len(L))
```

Comment:
Do we not support `shape` anymore?!

Reply:
https://github.com/pymc-devs/pymc3/blob/181c8ddeb0683199d7ea94bd898d820325ebdeaf/pymc3/distributions/distribution.py#L93-L94